src/component/buttonPanel/buttonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from '../button';
+import styles from './buttonPanel.module.css';
+
+const Calculator = () => (
+  <div className="mainCalc">
+    <div className={styles.button_row}>
+      <Button label="AC" />
+      <Button label="+/-" />
+      <Button label="%" />
+      <Button label="÷" />
+    </div>
+    <div className={styles.button_row}>
+      <Button label="7" />
+      <Button label="8" />
+      <Button label="9" />
+      <Button label="x" />
+    </div>
+    <div className={styles.button_row}>
+      <Button label="4" />
+      <Button label="5" />
+      <Button label="6" />
+      <Button label="-" />
+    </div>
+    <div className={styles.button_row}>
+      <Button label="1" />
+      <Button label="2" />
+      <Button label="3" />
+      <Button label="+" />
+    </div>
+    <div className={styles.button_row}>
+      <Button label="0" />
+      <Button label="." />
+      <Button label="=" />
+    </div>
+  </div>
+);
+
+// Export Calculator Component.
```

Comment:
#### [OPTIONAL]

- [ ] Kindly remove this comment, the export below clearly states you are exporting the component.